app-modules/in-app-communication/database/migrations/2025_12_15_121125_remove_twilio_chat_feature.php

```diff
@@ -34,27 +34,21 @@
 </COPYRIGHT>
 */
 
-namespace AidingApp\InAppCommunication\Actions;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+use Spatie\LaravelSettings\Migrations\SettingsMigration;
 
-use App\Models\User;
+return new class () extends SettingsMigration {
+    public function up(): void
+    {
+        Schema::dropIfExists('twilio_conversation_user');
+        Schema::dropIfExists('twilio_conversations');
+
+        $this->migrator->deleteIfExists('twilio.api_key');
```

Comment:
We haven't in the past, idk if we should mix settings migrations and schema migrations.

I also think that although the data is not recoverable in a reverse here, the tables and setting can be recreated.